Lib/ast.py

```diff
@@ -114,7 +114,11 @@ def _convert(node):
     return _convert(node_or_string)
 
 
-def dump(node, annotate_fields=True, include_attributes=False, *, indent=None):
+def dump(
+    node, annotate_fields=True, include_attributes=False,
+    *,
+    indent=None, show_empty=True,
```

Comment:
I was hoping we could jump to omitting them by default. `dump()` already changes regularly as new versions change the AST structure (e.g., 3.12 added `type_params` to classes and functions).

Reply:
I am 50/50 on that :(

First of all, it really helps to explore the ast structure (I use `ast.dump` a lot in debugging https://github.com/wemake-services/wemake-python-styleguide).
Second, I think that it is ok to add new stuff (there's not other way), but it is not good to remove stuff by default that people can find useful.

But, I can easily be persuaded on switching the default :)